<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMachineWindow</class>
 <widget class="QWidget" name="AboutMachineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>856</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>856</width>
    <height>391</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Подробности о станке</string>
  </property>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип станка</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="brand_line_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="brand_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Бренд станка</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="go_area_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </widget>
  <widget class="QLabel" name="list_reapirs_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>421</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Подробности о станке</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="type_line_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="order_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>История работ</string>
   </property>
  </widget>
  <widget class="QLabel" name="last_repair_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дата последнего ремонта</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_repair_line_edit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
